bin/_mocha

```diff
@@ -204,18 +204,6 @@ mocha.reporter(program.reporter);
 
 mocha.ui(program.ui);
 
-// load reporter
-
-try {
-  Reporter = require('../lib/reporters/' + program.reporter);
-} catch (err) {
-  try {
-    Reporter = require(program.reporter);
-  } catch (err) {
-    throw new Error('reporter "' + program.reporter + '" does not exist');
-  }
-}
```

Comment:
Checking if the reporters exist is still necessary. Ignoring those which are not found is not the way to go, since there is a reason why they have been specified.